### DISCUSSION TITLE
Steam Profile Picture Support

### DISCUSSION THREAD

**Discussion by johnnymoe** [Earu/EasyChat#117] [Ideas]
This would be a great function to have!

**Comment by Earu** [comment]
You mean in front of player names right ? 
I've been wanting to implement something like that for a while but it requires some heavy refactoring. In the meantime you can hover player names and it will show their avatars in the chatbox 👍

**Comment by johnnymoe** [comment]
That would be an awesome addition. This is one of the most powerful chatboxes I've ever used
